### ISSUE TITLE
[Feature]: AnimateLCM support?

### ISSUE THREAD

**Issue by 311-code** [continue-revolution/sd-forge-animatediff#1]
Thanks for working on this. I am wondering if you may be adding the new AnimateLCM stuff and in forge if it will work similar to this workflow? https://old.reddit.com/r/StableDiffusion/comments/1ajjx4t/animatelcm_support_for_comfyui_just_dropped/

Not sure if this needs something similar to the https://github.com/jojkaart/ComfyUI-sampler-lcm-alternative the better LCM sampler functions. I tried this workflow out made probably the best looking animatediff videos I've made before. Thanks!

**Comment by continue-revolution** [comment]
Yes I will support this. You can just send me the official link to AnimateLCM project. It will be supported in forge for the time being, and I will see how convenient/hard it is to implement it in OG A1111 given that maintaining sd-webui-controlnet is becoming notoriously challenging

**Comment by 311-code** [comment]
Awesome, no problem. Here is the model https://huggingface.co/wangfuyun/AnimateLCM/tree/main and info https://animatelcm.github.io/ I am actually not sure if it's just the model itself that is making the difference or the LCM alternative sampler.

**Comment by jtrozos** [comment]
Strangely It was working yesterday, had to click generate without the animatelcm lora (sd15_lora_beta) first, just stop it when the steps start, and then I could activate the lora sd15_lora_beta.safetensors, and it was working without errors.. similar to a1111 right now, i have to do that first run without the lora

**Comment by jtrozos** [comment]
Hi, Is there anyway to downgrade forge to February 23 state? I would love to keep using animatelcm in forge (faster than a1111) until it's officially fixed. Thanks!

**Comment by continue-revolution** [comment]
In your webui repo, do ‘git checkout \<commit hash\>’ in your terminal. You can find your favorite commit hash in the commit history. Since forge does not have a git tag, this is the only way that you can revert back to a previous version.

**Comment by continue-revolution** [comment]
I highly encourage anyone who likes to play with image generation to learn some basic git. Your life will be much easier if you are able to write some simple command, and prompt GPT for more complex use cases.

**Comment by jtrozos** [comment]
I did a 'git log' in forge folder to find the commit hash, but it's not showing any before day 23.. the earlier is Fri Feb 23 18:34:24 'fix resize-handle for vertical layout' but with that one, the fatal error is still happening.. when it was somehow working by not selecting the animatelcm lora on the first run..  is it possible to downgrade further than day 23? It's just to try.. It's great going 30% faster than a1111. Thanks!  (the fatal error is that one ending with 'Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions'
